spring-cloud-netflix-core/src/main/resources/META-INF/spring.factories

```diff
@@ -20,4 +19,10 @@ org.springframework.cloud.client.discovery.EnableDiscoveryClient=\
 org.springframework.cloud.netflix.eureka.EurekaDiscoveryClientConfiguration
 
 org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker=\
-org.springframework.cloud.netflix.hystrix.HystrixCircuitBreakerConfiguration
\ No newline at end of file
+org.springframework.cloud.netflix.hystrix.HystrixCircuitBreakerConfiguration
+
+org.springframework.cloud.netflix.metrics.servo.EnableServo=\
```

Comment:
Just to document from our slack conversation, I'm going to change `@Enable{Servo|Specator}` to auto configure with `@ConditionalOnMissingClass(Registry.class)` for Servo and `@ConditionalOnClass(Registry.class)` for Spectator.